### PR TITLE
Feature/#23 process breed

### PR DESCRIPTION
**Pull request by Enessetere**
Fix dice roll, change dependency for player's herd, cleanup and breedProcessor up and living.

Tests for random elements are confusing, so will be added later with code refactor.

PR for unblock GameController.

### CODE REVIEW

**Review by SebastianBabinski1 — APPROVED** [review]
Looks good, but can you add some basic comments, because now we need to think about how the functionality works? :)

File: src/app/Dice.ts [hunk 1]
```diff
@@ -5,44 +5,6 @@ import { sample } from 'lodash';
...
-      AnimalNames.RABBIT,
-      AnimalNames.RABBIT,
-      AnimalNames.SHEEP,
-      AnimalNames.SHEEP,
-      AnimalNames.SHEEP,
-      AnimalNames.PIG,
-      AnimalNames.COW,
-      AnimalNames.WOLF,
-    ]);
+    return sample(this.diceSides) as AnimalNames;
```
Comment (synowa) [inline src/app/Dice.ts:8]: yeah, thanks for spotting that bug!

File: src/app/BreedProcessor.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  }
+
+  private breedAnimals(
+    foxDiceAnimalName: AnimalNames | undefined,
+    wolfDiceAnimalName: AnimalNames | undefined,
+    herd: Herd,
+  ) {
+    if (
+      foxDiceAnimalName &&
+      wolfDiceAnimalName &&
```
Comment (synowa) [inline src/app/BreedProcessor.ts:64]: You don't need the line 64 :) 63 and 65 will work the same without it.

**Review by synowa — APPROVED** [review]
I tried to use it and it worked. Good job 😄

**Pull request merged**